addons/web/static/src/js/fields/field_registry.js

```diff
@@ -21,7 +22,7 @@ registry
     .add('abstract', AbstractField)
     .add('input', basic_fields.InputField)
     .add('integer', basic_fields.FieldInteger)
-    .add('boolean', basic_fields.FieldBoolean)
+    .add('boolean', basic_fields_owl.FieldBoolean)
```

Comment:
maybe we should have a side registry, for converted fields, to be fully backward compatible